packages/storybook/package.json

```diff
@@ -23,6 +24,7 @@
   },
   "scripts": {
     "storybook": "start-storybook -p 6006",
-    "build": "build-storybook -output-dir ./dist"
+    "build": "build-storybook -o ./dist",
+    "chromatic": "npx chromatic --project-token"
```

Comment:
This is how we start a remote test run locally but not used explicitly in CI because we have the github action plugin yes? Also why `npx` here? Does the package not ship with a CLI executable for us?

Reply:
> This is how we start a remote test run locally correct? And this script is not used explicitly in CI because we have the github action plugin?

Correct.

> Also why npx here? Does the package not ship with a CLI executable for us?

Honestly, I'm not sure. I lifted this line [from their docs](https://www.chromatic.com/docs/setup#run-chromatic), I'm happy to dig further.